gradoop-flink/src/main/java/org/gradoop/flink/algorithms/gelly/shortestpaths/functions/SingleSourceShortestPathsAttribute.java

```diff
@@ -40,8 +42,7 @@ public SingleSourceShortestPathsAttribute(String shortestPathProperty) {
   }
 
   @Override
-  public EPGMVertex join(org.apache.flink.graph.Vertex<GradoopId, Double> gellyVertex,
-    EPGMVertex gradoopVertex) {
+  public V join(org.apache.flink.graph.Vertex<GradoopId, Double> gellyVertex, V gradoopVertex) {
```

Comment:
We should name it vertex instead of gradoopVertex or generify all namings from the other classes where gelly and gradoop vertices are both used in arguments.